2021-07-17 12:46:12,912 INFO    MainThread:2444 [wandb_setup.py:_flush():69] setting env: {}
2021-07-17 12:46:12,912 INFO    MainThread:2444 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-17 12:46:12,913 INFO    MainThread:2444 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\Qiandehou\Desktop\github\pytorch_and_something\SIIM_FISABIO_RSNA_COVID_19_Detection\wandb\run-20210717_124612-ln42ztwo\logs\debug.log
2021-07-17 12:46:12,913 INFO    MainThread:2444 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\Qiandehou\Desktop\github\pytorch_and_something\SIIM_FISABIO_RSNA_COVID_19_Detection\wandb\run-20210717_124612-ln42ztwo\logs\debug-internal.log
2021-07-17 12:46:12,915 INFO    MainThread:2444 [wandb_init.py:init():370] calling init triggers
2021-07-17 12:46:12,917 INFO    MainThread:2444 [wandb_init.py:init():375] wandb.init called with sweep_config: {}
config: {}
2021-07-17 12:46:12,917 INFO    MainThread:2444 [wandb_init.py:init():390] re-initializing run, found existing run on stack: 26c3313t
2021-07-17 12:46:12,917 INFO    MainThread:2444 [wandb_run.py:finish():1293] finishing run poi/pytorch_and_something/26c3313t
2021-07-17 12:46:12,918 INFO    MainThread:2444 [jupyter.py:save_history():339] not saving jupyter history
2021-07-17 12:46:12,918 INFO    MainThread:2444 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-17 12:46:12,918 INFO    MainThread:2444 [wandb_init.py:_jupyter_teardown():269] cleaning up jupyter logic
2021-07-17 12:46:12,918 INFO    MainThread:2444 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 0
2021-07-17 12:46:12,919 INFO    MainThread:2444 [wandb_run.py:_restore():1565] restore
2021-07-17 12:46:18,599 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:46:23,716 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:46:28,837 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:46:33,956 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:46:39,067 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:46:44,181 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:46:49,302 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:46:54,421 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:46:59,540 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:47:04,660 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:47:09,778 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:47:14,894 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:47:20,009 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
2021-07-17 12:47:25,126 INFO    MainThread:2444 [wandb_run.py:_wait_for_finish():1715] got exit ret: None
